src/java.desktop/share/classes/java/awt/geom/Path2D.java

```diff
@@ -292,6 +292,9 @@ public Float(Shape s, AffineTransform at) {
             }
         }
 
+        /**
+         * @since 10
+         */
```

Comment:
Not sure it's required…

If it is, you should also add explicit `{@inheritDoc}`:

```suggestion
        /**
         * {@inheritDoc}
         *
         * @since 10
         */
```

Reply:
1. You don't _need_ an explicit `{@inheritDoc}` although you may choose to do so for explicit clarity.

2. `@since` can be omitted for a member if the value would be the same as on the enclosing class

3. When present, `@since` on a method should indicate the first release in which the method is available for use on that class, with that VM signature (includes args and return type)

Reply:
Thank you for clarification, @jonathan-gibbons. So, `@since` is required here.

I prefer an explicit `{@inheritDoc}`, this way the javadoc comment doesn't look empty. I'm fine without adding `{@inheritDoc}` though.